bundles/ranvier-commands/commands/drop.js

```diff
@@ -32,7 +32,8 @@ module.exports = (srcPath) => {
         npc.emit('playerDropItem', player, item);
       }
 
-      Broadcast.sayAt(player, `<green>You dropped: </green>${item.display}<green>.</green>`);
+      const formatDropMessage = (name, item) => `<green>${name} dropped: </green>${item.display}<green>.</green>`;
```

Comment:
Nope, it's a relic from when I had it being broadcast to the room (which could be spammy). I could just remove that, but I wonder if the stuff prone to abuse could still be broadcast to party members in the room?